rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+newline_style = "Unix"
+edition = "2018"
+version = "One"
```

Comment:
I'd like to specify `version`, but it's not stable.  If you remove it and reformat src/lib.rs, this PR should be good to go.

Reply:
Fair enough! but not sure why src/lib.rs only. For #204 don't we need to reformat all?

Reply:
Sorry it wasn't clear.  I didn't mean to remove formatting for everything else; I meant that src/lib.rs needed to be reformatted on top of the changes already made in the pull request.  When I checked out this branch and ran `cargo fmt`, everything was clean except src/lib.rs

Reply:
Oh, thanks for the clarification :)